content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -7,200 +7,250 @@ section: doc
-
-image::tutorials/IBMCloud/worker-pool.png[]
-
-* Choose **Master Service Endpoint**,  In VRF-enabled accounts, you can choose private-only to make your controller accessible on the private network or via VPN tunnel. Choose public-only to make your controléer publicly accessible. When you have a VRF-enabled account, your cluster is set up by default to use both private and public endpoints. For more information visit https://cloud.ibm.com/docs/account?topic=account-service-endpoints-overview[endpoints].
-
-image::tutorials/IBMCloud/endpoints.png[]
-
-* Give cluster a **name**
-
-image::tutorials/IBMCloud/name-new.png[]
-
-* Give desired **tags** to your cluster, for more information visit https://cloud.ibm.com/docs/account?topic=account-tag[tags]
-
-image::tutorials/IBMCloud/tasg-new.png[]
-
-* Click **create**
-
-image::tutorials/IBMCloud/create-new.png[]
-
-* Wait for your cluster to be provisioned
-
-image::tutorials/IBMCloud/cluster-prepare.png[]
-
-* Your cluster is ready for usage
-
-image::tutorials/IBMCloud/cluster-done.png[]
-
-## Step 2 deploy IBM Cloud Block Storage plug-in
+To deploy Jenkins on IBM Cloud:
+
+. <<Provisioning a Kubernetes cluster,Provision a Kubernetes Cluster>>.
+If you already have a Kubernetes cluster, you can skip to step 2.
+. <<Deploying the IBM Cloud Block Storage plug-in,Deploy IBM Cloud Block Storage plug-in>>.
+If you have already deployed the the IBM Cloud Block Storage plug-in, you can skip to step 3.
+. <<Deploying Jenkins,Deploy Jenkins>>.
+. <<Verifying the Jenkins installation,Verify the Jenkins installation>>.
+
+== Provisioning a Kubernetes cluster
+
+When deploying Jenkins to IBM Cloud, you must first provision a Kubernetes cluster.
+
+NOTE: If you already have a Kubernetes cluster, you can skip to <<Deploying the IBM Cloud Block Storage plug-in>>.
+
+To provision a Kubernetes cluster:
+
+. Select *Catalog* in the upper left corner.
+. Select *Service* from the catalog.
+. Search for *Kubernetes Service* and select it.
++
+.Kubernetes search screen
+image::tutorials/IBMCloud/kubernetes-select.png[IBM Kubernetes selection,role="screenshot"]
++
+You are now at the Kubernetes deployment page, and you need to specify some details about the cluster.
+
+. Choose a plan type of *standard* or *free*.
+The free plan only has one worker node and no subnet.
+. To provision a standard cluster, you will need to upgrade your account to Pay-As-You-Go.
+To upgrade to a Pay-As-You-Go account, complete the following steps:
+.. In the console, go to *Manage > Account*
+.. Select *Account settings*, and then select *Add credit card*.
+.. Enter your payment information, select *Next*, and then submit your information.
+. Choose *Classic* or *VPC*, read the link:https://cloud.ibm.com/docs/containers?topic=containers-infrastructure_providers[docs], and then choose the most suitable type for yourself.
++
+image::tutorials/IBMCloud/infra-select.png[IBM infrastructure selection]
```

Comment:
Some images now include the figure references, but others don't. I think we should either include them for all images or not include them for any. Since it doesn't appear the other Jenkins topics include them, you could remove them for now, so they're consistent?

Reply:
Went with full removal to have everything be consistent with the other tutorial pages. If this is something that ends up being added later to another page, I can always come back and update things.